diff --git a/dist/oauth-Dtt-caL2.d.mts b/dist/oauth-Dtt-caL2.d.mts
index 7dc7a3b92fe1a7df8a8561992059ba37f9668d50..b0713ac61d38c55562452c3e0b7661aaec6f22e0 100644
--- a/dist/oauth-Dtt-caL2.d.mts
+++ b/dist/oauth-Dtt-caL2.d.mts
@@ -110,7 +110,6 @@ declare function buildOAuthDeviceGrant(): {
     resource: {
       oneOf: ({
         type: string;
-        items?: undefined;
       } | {
         type: string;
         items: {
@@ -786,9 +785,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
             required: false;
             schema: {
               type: "string";
-              format?: undefined;
-              minimum?: undefined;
-              items?: undefined;
             };
             description: string;
           } | {
@@ -797,9 +793,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
             required: true;
             schema: {
               type: "string";
-              format?: undefined;
-              minimum?: undefined;
-              items?: undefined;
             };
             description: string;
           } | {
@@ -809,8 +802,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
             schema: {
               type: "string";
               format: string;
-              minimum?: undefined;
-              items?: undefined;
             };
             description: string;
           } | {
@@ -820,8 +811,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
             schema: {
               type: "integer";
               minimum: number;
-              format?: undefined;
-              items?: undefined;
             };
             description: string;
           } | {
@@ -833,8 +822,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
               items: {
                 type: "string";
               };
-              format?: undefined;
-              minimum?: undefined;
             };
             description: string;
           })[];
@@ -1073,7 +1060,6 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
                       oneOf: ({
                         type: string;
                         description: string;
-                        items?: undefined;
                       } | {
                         type: string;
                         items: {
@@ -1391,12 +1377,10 @@ declare const oauthProvider: <O extends OAuthOptions<Scope[]>>(options: O) => {
         allowedMediaTypes: string[];
         openapi: {
           description: string;
-          security: ({
-            bearerAuth: never[];
-            OAuth2?: undefined;
-          } | {
-            OAuth2: string[];
-            bearerAuth?: undefined;
+            security: ({
+              bearerAuth: never[];
+            } | {
+              OAuth2: string[];
           })[];
           parameters: {
             name: string;
